laurent22/joplin
# Desktop: Rich text editor: Mermaid diagram download button doesn't work

This PR fixes #9588 with Mermaid code blocks in the rich text editor where an unnecessary "Download" button appears when 

Changes Made:

Modified the Mermaid rendering rule in packages/renderer/MdToHtml/rules/mermaid.ts
Removed the code that adds the "Download" button to Mermaid diagrams
Ensured that the graph still renders correctly without any visual artifacts

Testing:


https://github.com/user-attachments/assets/96f2f186-447b-41df-8987-2a38ae9f4dd9



Steps to Test:

Create a new note and switch to the rich text editor
Add a code block with language mermaid
Set the following content:
```
CopygitGraph
    commit
    commit
```

Click "Ok" to render the graph
Hover over the graph and verify that no "Download" button appears
 let me know if any further changes or information are needed.
